topic_manager.go

```diff
@@ -31,14 +31,18 @@ type TopicManager interface {
 }
 
 type topicManager struct {
-	brokers            []string
-	broker             Broker
+	admin              sarama.ClusterAdmin
 	client             sarama.Client
 	topicManagerConfig *TopicManagerConfig
 }
 
 // NewTopicManager creates a new topic manager using the sarama library
 func NewTopicManager(brokers []string, saramaConfig *sarama.Config, topicManagerConfig *TopicManagerConfig) (TopicManager, error) {
+
+	if !saramaConfig.Version.IsAtLeast(sarama.V0_10_0_0) {
+		return nil, fmt.Errorf("goka's topic manager needs kafka version v0.10.0.0 or higher to function. Version is %s", saramaConfig.Version.String())
```

Comment:
Question: do we need v0.10.0.0 now? is this a breaking change?

Reply:
Goka requires kafka version `v0.10.2.0` since version v1.0.0, as the consumer-group interface requires this. I guess we could check that in advance, but sarama will fail after creating the first processor anyway.